### PR TITLE
Update rulesets: NYDailyNews

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/NYDailyNews.xml [hunk 1]
```diff
@@ -70,29 +72,27 @@ Fetch error: http://nydailynews.com/ => http://nydailynews.com/: (60, 'SSL certi
...
+	<target host="m.nydailynews.com" />
+	<target host="myaccount2.nydailynews.com" />
+	<target host="placeanad.nydailynews.com" />
+	<target host="static1.nydailynews.com" />
+	<target host="static2.nydailynews.com" />
+	<target host="static3.nydailynews.com" />
+	<target host="webport.nydailynews.com" />
 
+	<!-- Used on classifieds.nydailynews.com -->
+	<target host="dmpwow64jb5ov.cloudfront.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/NYDailyNews.xml]: Please move this to `Cloudfront.xml`

File: src/chrome/content/rules/NYDailyNews.xml [hunk 2]
```diff
@@ -70,29 +72,27 @@ Fetch error: http://nydailynews.com/ => http://nydailynews.com/: (60, 'SSL certi
...
 
 	<securecookie host="^[^@:/]*\.nydailynews\.com$" name=".+" />
 
-
-	<rule from="^http://(?:static\d|www)\.nydailynews\.com/"
+	<rule from="^http://static\d\.nydailynews\.com/"
```
Comment (J0WI) [inline src/chrome/content/rules/NYDailyNews.xml]: Please add some test urls for this.

Comment (injust) [reply src/chrome/content/rules/NYDailyNews.xml]: I actually can't find any usage of `static[1-3].nydailynews.com` anymore, so going to remove the rule instead.

File: src/chrome/content/rules/NYDailyNews.xml (hunk 1 above)
Comment (injust) [reply src/chrome/content/rules/NYDailyNews.xml]: Fixed.

File: src/chrome/content/rules/NYDailyNews.com-falsemixed.xml [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
 	<target host="nydailynews.com" />
 	<target host="www.nydailynews.com" />
 
+	<rule from="^http://nydailynews\.com/"
+		  to="https://www.nydailynews.com/" />
+
+	<exclusion pattern="^http://(www.)?nydailynews\.com/(autos|horoscopes)" />
+		<test url="http://www.nydailynews.com/autos" />
+		<test url="http://www.nydailynews.com/autos/" />
+		<test url="http://www.nydailynews.com/horoscopes" />
```
[the remark this answers is not in the document]
Comment (injust) [reply src/chrome/content/rules/NYDailyNews.com-falsemixed.xml]: Fixed it, thanks.
Just noticed that I had an unescaped `.` in the pattern anyway, oops 😛

File: src/chrome/content/rules/NYDailyNews.xml [hunk 4]
```diff
@@ -26,6 +26,7 @@
 		- live.nydailynews.com: Works; mismatched, CN: *.scribblelive.com
 		- liveblog.nydailynews.com: Works; mismatched, CN: *.scribblelive.com
 		- local.nydailynews.com: Works; mismatched, CN: *.localedge.com
+		- m: Invalid certificate
```
Comment (J0WI) [inline src/chrome/content/rules/NYDailyNews.xml]: Please also remove the `Fully covered subdomains` section.

**Pull request merged**